packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -82,12 +81,12 @@ export class EncryptionManager extends Encryption {
         return storedConnection;
     }
 
-    public encryptConnection(connection: Connection): StoredConnection {
+    public encryptConnection(connection: Omit<Connection, 'created_at' | 'updated_at'>): Omit<StoredConnection, 'created_at' | 'updated_at'> {
```

Comment:
this is bit sad, we are not passing complete Connection so I had to deal the way I can